code-quality/ci-setup/get-started.md

```diff
@@ -4,11 +4,11 @@ description: Automate your code quality enforcement with just a few clicks.
 
 # GitHub Integration
 
-Trunk Code Quality has the ability to post its results to the [Trunk Code Quality web app](https://app.trunk.io/login?intent=check). This will enable you to view your repository's Code Quality history over time so you can track the trend of issues in your code, as well as browse the issues in your repository to help you understand which issues should be prioritized to fix.
+Trunk Code Quality has the ability to post its results to the [Trunk Code Quality web app](https://app.trunk.io/login?intent=code+quality). This will enable you to view your repository's Code Quality history over time so you can track the trend of issues in your code, as well as browse the issues in your repository to help you understand which issues should be prioritized to fix.
 
 ## Connect your Trunk organization to GitHub
 
-Sign up at [app.trunk.io](https://app.trunk.io), create a Trunk organization, and connect it to your repositories. You will need to grant the following [GitHub App permissions](../../administration/github-app-permissions.md).
+Sign up at [app.trunk.io](https://app.trunk.io/?intent=code+quality), create a Trunk organization, and connect it to your repositories. You will need to grant the following [GitHub App permissions](../../administration/github-app-permissions.md).
```

Comment:
I don't think `https://app.trunk.io/?intent=code+quality` does anything

Reply:
Yeah, weird.  I've changed them all to have the /login/